src/Logging.Legacy/LegacyOrleansLoggingFactoryExtensions.cs

```diff
@@ -55,8 +55,8 @@ public static class LegacyOrleansLoggingFactoryExtensions
         }
 
         /// <summary>
-        /// Add event bulking feature onto <param name="provider"></param>, and add that new logger provider 
-        /// <see cref="EventBulkingLoggerProvider{TDecoratedLoggerProvider}"/> into <param name="builder"></param>.
+        /// Add event bulking feature onto <paramref name="provider"></paramref>, and add that new logger provider 
```

Comment:
paramref can be self closing here I think.

Reply:
yes it can. But is that preferred? based on which guideline? I'd love to follow conventions as long as I have a guideline to follow. especially when next time I'm writing it, I will know what thing is better to do .

Reply:
I prefered self-closing for any tag which is empty.